### PR TITLE
Add debug mode

### PR DESCRIPTION
**Pull request by fernando457829**

### CODE REVIEW

**Review by Doges — APPROVED** [review]
LGTM

File: index.js [hunk 1]
```diff
@@ -23,7 +23,14 @@ const CLIENT_OPTIONS = {
 
 console.log(readFileSync('bigtitle.txt', 'utf8').toString())
 
+const debug = Boolean(process.env.DEBUG)
```
Comment (metehus) [inline index.js:26]:
```suggestion
const debug = process.env.DEBUG === 'true'
```


image

**Review by metehus — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (fernando457829) [reply index.js:26]: WTF, I forgot that DotEnv don't transform booleans

**Review by Doges — APPROVED** [review]
LGTM

**Pull request merged**